examples/binary/README.md

```diff
@@ -13,8 +13,14 @@ repo:
   permissions:
     "*":
       - "*"
-```
+</code>
 
-You can send HTTP PUT/GET requests
-to `http://localhost:8080/repo/<filename>` to upload/download a binary file,
-e.g. `http://localhost:8080/repo/libsqlite3.so`.
+Use `PUT` HTTP request to upload a file, and `GET` for downloading.</p>
+
+Example using [HTTPie](https://httpie.org/) CLI tool for uploading and downloading:
+<code>
+http -a {{user}}:password PUT https://central.artipie.com/{{user}}/{{name}}/file.bin @file.bin
+http GET https://central.artipie.com/{{user}}/{{name}}/file.bin --output=./file.bin
+</code>
+
+// TODO: add curl example
```

Comment:
@g4s8 hm, markdown looks terrible now
![image](https://user-images.githubusercontent.com/14931449/87780311-1fc54580-c837-11ea-9461-cead0b3f7cd4.png)
I think we should not leave it like this